receiver/opencensusreceiver/opencensus.go

```diff
@@ -164,34 +155,47 @@ func (ocr *Receiver) grpcServer() *grpc.Server {
 	return ocr.serverGRPC
 }
 
-// StopTraceReception is a method to turn off receiving traces. It
-// currently is a noop because we don't yet know if gRPC allows
-// stopping a specific service.
+// StopTraceReception is a method to turn off receiving traces. It stops
+// metrics reception too.
 func (ocr *Receiver) StopTraceReception() error {
-	// StopTraceReception is a noop currently.
 	// TODO: (@odeke-em) investigate whether or not gRPC
```

Comment:
Is this TODO still needed?

Reply:
I don't think that in practice we need that, so far we lived without it. Removing.